v2/components/introVideo.jsx

```diff
@@ -0,0 +1,37 @@
+import { breakpoints } from '../assets/styles/constants';
+
+const IntroVideo = () => (
+  <div className="intro-video">
+    <iframe
+      title="Intro video"
+      src="https://arteveldehogeschool.cloud.panopto.eu/Panopto/Pages/Embed.aspx?id=dc47c1a8-68b9-413b-812a-aa1400a18754&v=1"
+      frameBorder="0"
+      allowFullScreen
+      allow="autoplay"
+    />
```

Comment:
The video player does not look the same as on the original version - specifically the fit of the video. I'd copy this from v1